### DISCUSSION TITLE
Vendors

### DISCUSSION THREAD

**Discussion by luni64** [TeensyUser/doc#12] [General]
Hi @FrankBoesing, 
any special reason why you deleted [ExpTech ](https://www.exp-tech.de/plattformen/teensy/)from the list of German vendors? Actually this is one of my favorite vendors  so I'm a bit surprised...

**Comment by FrankBoesing** [comment]
Hi,

ja, die liefern seit April nicht mehr an Privatkunden. B2B only.

**Comment by luni64** [comment]
Danke, ist mir noch gar nicht aufgefallen.  Das ist bei deren Sortiment jedenfalls eine bizarre Entscheidung.
